superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberTotal/transformProps.ts

```diff
@@ -38,6 +41,7 @@ export default function transformProps(chartProps: BigNumberTotalChartProps) {
     timeFormat,
     yAxisFormat,
   } = formData;
+  const refs: Refs = {};
```

Comment:
the `refs` object is needed here so the downstream code can add the reference to the chart div for the tooltip positioning callback to calculate the current location of the pointer.